csv2rdf/csv2rdf_single_subject.py

```diff
@@ -0,0 +1,75 @@
+import csv
+import validators
+import sys
+import json
+import os
+from rdflib import Graph, URIRef, Literal
+from rdflib.namespace import RDF
+from typing import Optional, List
+
+DIRNAME = os.path.dirname(__file__)
+mapping_filename = os.path.join(DIRNAME, 'mapping.json')
+dest_filename = os.path.join(DIRNAME, 'out_rdf.ttl')
+
+def convert_csv_to_turtle(filenames: List[str]) -> Graph:
+    """
+    (List[str]) -> Graph
+
+    Adds all informations as RDF triples from the input filenames into a graph and return it.
+    *Important: Each input file must have the first column as subjects of all triples
+    
+    @Pre: type(filenames) == List[str]
+    """
+    g = Graph()
+
+    ontology_dict = json.load(open(mapping_filename, "r"))
+
+    for filename in filenames:
+        with open(filename, "r", encoding="utf-8") as csv_file:
+            csv_reader = csv.reader(csv_file)
+
+            header = next(csv_reader)
+            header_without_subject = header[1:]
+            predicates = []
+            for column in header_without_subject:
+                if column in ontology_dict:
+                    predicates.append(URIRef(ontology_dict[column]))
+
+            try:
```

Comment:
This should be a call to the dictionary's `get` method.

Reply:
I changed the logic here.